PingCAP-QE/Naglfar
# TestClusterTopology: prune TombStone tikv automatically

### What problem does this PR solve?

Issue Number: close #91  

Problem Summary:  tikv tombstone resources need to be actively pruned

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

How it Works:
In ready(testclustertopology) state, reconcile check tidb cluster by calling cmd(tiup cluster display).

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)
1.tikv(n4,n3,n2,n1)--> tikv (n3,n2,n1)
check n4 tikv is pruned successfully
2.tikv(n3,n2,n1)--> tikv (n2,n1)
check reconcile if try prune continuously
3.tikv(n2,n1)--->tikv(n2,n1,n4)
check n3 tikv is pruned successfully

Side effects

- Breaking backward compatibility
